docs/guides/vpn/dual-VPN.md

```diff
@@ -29,28 +24,28 @@ And make sure that the DNS requests go though the instance of OpenVPN:
 push "dhcp-option DNS 10.9.0.1"
 ```
 
-One other setting that we need to change is comment out (the already existing line) `# push "redirect-gateway def1 bypass-dhcp"`.
-Save the file and run the second instance of VPN:
+One other setting that we need to change is to comment out `# push "redirect-gateway def1 bypass-dhcp"`.
+Save the file and run the second instance of OpenVPN:
 
 ```
 systemctl start openvpn@server2.service
 ```
 
-If your distro doesn’t have “systemctl” you may use commands like below to start your OpenVPN with your second configuration as a daemon: 
+If the distro doesn’t have `systemctl` you may use commands like below to start your OpenVPN with your second configuration as a daemon: 
```

Comment:
Sorry, I meant the other first "your" (right before "OpenVPN") :sweat_smile:. Sorry!